Modules/Quotas/test/Quotas.Domain.Tests/Tests/Tiers/TierTests.cs

```diff
@@ -187,12 +186,12 @@ public void AddQuotaForAllMetricsOnQueuedForDeletion_only_creates_missing_quotas
     }
 }
 
-
 file static class TierExtensions
 {
     public static Tier Clone(this Tier tier)
     {
-        var serialized = JsonConvert.SerializeObject(tier);
-        return JsonConvert.DeserializeObject<Tier>(serialized)!;
+        var newTier = new Tier(tier.Id, tier.Name);
+        newTier.Quotas.AddRange(tier.Quotas);
```

Comment:
We don't modify entities directly. Only with the help of methods.

Was there something wrong with the json serialization+deserialization?

Reply:
It used `Newtonsoft.Json` and couldn't clone everything, which caused some errors. I now replaced it with the .NET serialization, which now works fine.

Reply:
Okay, cool 👍